scram-tools.file/tools/jemalloc-prof/jemalloc-prof.xml

```diff
@@ -0,0 +1,10 @@
+<tool name="jemalloc-debug" version="@TOOL_VERSION@">
```

Comment:
@gartung , please fix the tool file. The name should be `jemalloc-prof` and we should have `<lib name="jemalloc-prof"/>`